integration-cli/environment/clean.go

```diff
@@ -210,12 +208,3 @@ func getAllPlugins() (types.PluginsListResponse, error) {
 	}
 	return plugins, nil
 }
-
-// DaemonHost return the daemon host string for this test execution
-func DaemonHost() string {
```

Comment:
Moving this to request as it makes a little more sense and otherwise, it's an import cycle…